engine/src/main/scala/cromwell/webservice/metadata/MetadataBuilderActor.scala

```diff
@@ -172,17 +172,17 @@ object MetadataBuilderActor {
     } mapValues { md => JsArray(md.toVector.sortBy(_.index) flatMap { _.metadata }) }
 
     val wrappedCalls = JsObject(Map(WorkflowMetadataKeys.Calls -> JsObject(callsMap)))
-
-    JsObject(foldedWorkflowValues.fields ++ wrappedCalls.fields)
+    val callData = if (callsMap.isEmpty && !includeCallsIfEmpty) Nil else wrappedCalls.fields
```

Comment:
I don't think so, this prevents from having `"calls": {}` in some cases if I understand which doesn't affect the empty stuff